crates/governance/src/vp/pgf.rs

```diff
@@ -207,33 +212,193 @@ where
             },
         )
     }
+
+    /// Validate a pgf balance change
+    pub fn is_valid_balance_change(
+        _ctx: &'ctx CTX,
+        _token: &Address,
+    ) -> Result<()> {
+        Err(Error::new_alloc(
```

Comment:
Same here, could be a `new_const`. Maybe we don't even need this function, could just return the error directly when matching on the key but it's just a personal preference